packages/components/src/tab-panel/README.md

```diff
@@ -38,35 +84,35 @@ const MyTabPanel = () => (
 );
 ```
 
-## Props
+### Props
 
 The component accepts the following props:
 
-### className
+#### className
```

Comment:
@melchoyce — should these be H4s (like the are currently), or should they just be bold? I see you're using bold in #13320.

Reply:
I'm not sure — what did we use in the other doc update PRs? I can change mine to match if we need.

Reply:
Looks like we've been inconsistent: 

H4's: 
https://github.com/WordPress/gutenberg/pull/11226
https://github.com/WordPress/gutenberg/pull/12485

Bold Text: 
https://github.com/WordPress/gutenberg/pull/11872

Reply:
Let's go with H4s, looks like it's just me being inconsistent.